Source/Morris.Reducible/Reducer.cs

```diff
@@ -23,10 +23,9 @@ public static partial class Reducer
 		return (state, delta) =>
 		{
 			bool anyChanged = false;
-			for (int o = 0; o < allReducers.Length; o++)
+			foreach (var reducer in allReducers)
 			{
-				(bool changed, state) = allReducers[o](state, delta);
-				anyChanged |= changed;
+				anyChanged |= reducer(state, delta).Changed;
```

Comment:
You've introduced a bug here as you are not doing anything with the returned reducer state.